### DISCUSSION TITLE
Do spotify jams work?

### DISCUSSION THREAD

**Discussion by givskun** [SpotCompiled/SpotilifeC#46] [Q&A]
does anyone know if they work and if so how?

**Comment by yodaluca23** [comment]
For one, this repo, SpotilifeC is not supported anymore, I suggest 
https://github.com/SpotCompiled/SpotveeC
 Or 
https://github.com/SpotCompiled/SpotC-Plus-Plus
Which are much more updated, and have more features.
Two, I believe in person jams work, but not remote.

**Reply by givskun** [reply]
thanks <3
